sql/src/main/java/com/vmware/ddlog/util/sql/CalciteToH2.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018-2020 VMware, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: BSD-2
+ */
+
+package com.vmware.ddlog.util.sql;
+
+import org.apache.calcite.schema.ColumnStrategy;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.ddl.SqlColumnDeclaration;
+import org.apache.calcite.sql.ddl.SqlCreateTable;
+import org.apache.calcite.sql.util.SqlBasicVisitor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+// We just need to strip out the subtype for the array, which H2 doesn't like.
```

Comment:
Here you should add a comment about what this class is doing: generates a SQL string that complies with some other rules. This comment should be instead in the body of the method that does this particular transformation, probably right at the statement where that is done.

Reply:
Also, please use JavaDoc, for people who use IDEs it is very useful.